assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -0,0 +1,134 @@
+const UPDATE_VALUE = 'UPDATE-VALUE';
```

Comment:
Did you think about future support for `ctrl` + `up`/`down`?

Reply:
Nope. what for? Scrubbing + what? All the point of using scrubbing is preventing the need of pressing arrows (up/down). I think that it become to complex, scrubbing is for making day work intuitive and fun. Thats my opinion.

Reply:
I talked about the number increasing mechanism (+1, +10, etc)
Let's talk about it.